core/rawdb/schema.go

```diff
@@ -105,7 +105,8 @@ var (
 
 	// Scroll L1 message store
 	syncedL1BlockNumberKey            = []byte("LastSyncedL1BlockNumber")
-	l1MessagePrefix                   = []byte("l1") // l1MessagePrefix + queueIndex (uint64 big endian) -> L1MessageTx
+	l1MessageLegacyPrefix             = []byte("l1")
+	l1MessagePrefix                   = []byte("L1") // l1MessagePrefix + queueIndex (uint64 big endian) -> L1MessageTx
```

Comment:
syndtr/goleveldb/leveldb library in Go is case-sensitive by default, So this is ok. 

```go
package main

import (
	"fmt"
	"github.com/syndtr/goleveldb/leveldb"
)

func main() {
	db, err := leveldb.OpenFile("path/to/db", nil)
	if err != nil {
		panic(err)
	}
	defer db.Close()
	err = db.Put([]byte("key"), []byte("value"), nil)
	if err != nil {
		panic(err)
	}

	data, err := db.Get([]byte("key"), nil)
	if err != nil {
		panic(err)
	}
	fmt.Println(string(data))

	data1, err := db.Get([]byte("Key"), nil)
	if err != nil {
		panic(err)
	}
	fmt.Println(string(data1))
}
```

```
value
panic: leveldb: not found
```